Geta.Optimizely.Sitemaps.sln

```diff
@@ -7,9 +7,35 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Sandbox", "Sandbox", "{9003
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Geta.Optimizely.Sitemaps", "src\Geta.Optimizely.Sitemaps\Geta.Optimizely.Sitemaps.csproj", "{A56D25DD-73FB-4754-B054-C5CD9B52804F}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Geta.Optimizely.Sitemaps.Commerce", "src\Geta.Optimizely.Sitemaps.Commerce\Geta.Optimizely.Sitemaps.Commerce.csproj", "{39B5430D-35AF-4413-980B-1CE51B367DC7}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Geta.Optimizely.Sitemaps.Commerce", "src\Geta.Optimizely.Sitemaps.Commerce\Geta.Optimizely.Sitemaps.Commerce.csproj", "{39B5430D-35AF-4413-980B-1CE51B367DC7}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Foundation", "sandbox\Foundation\src\Foundation\Foundation.csproj", "{82A14BA5-4A85-4DC3-833E-37EBC47BB891}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Foundation", "sandbox\Foundation\src\Foundation\Foundation.csproj", "{82A14BA5-4A85-4DC3-833E-37EBC47BB891}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "SolutionItems", "SolutionItems", "{A71AFC18-7B1C-422E-B556-A1F124C79E34}"
+	ProjectSection(SolutionItems) = preProject
+		.gitignore = .gitignore
+		CHANGELOG.md = CHANGELOG.md
+		CONTRIBUTING.md = CONTRIBUTING.md
+		LICENSE.md = LICENSE.md
+		NuGet.config = NuGet.config
+		README.md = README.md
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "docs", "docs", "{FEDCD604-4393-4662-BD63-0469772CF527}"
+	ProjectSection(SolutionItems) = preProject
+		docs\editor-guide.md = docs\editor-guide.md
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "images", "images", "{FE02F591-778A-4883-BA52-DA2B6B93D483}"
+	ProjectSection(SolutionItems) = preProject
+		docs\images\docker-debugging-output.PNG = docs\images\docker-debugging-output.PNG
+		docs\images\docker-output.PNG = docs\images\docker-output.PNG
+		docs\images\sitemap-property.PNG = docs\images\sitemap-property.PNG
+		docs\images\sitemap-scheduled-job.PNG = docs\images\sitemap-scheduled-job.PNG
+		docs\images\SitemapAdd.png = docs\images\SitemapAdd.png
+		docs\images\SitemapAddMultiSite.png = docs\images\SitemapAddMultiSite.png
+		docs\images\SitemapConfigure.png = docs\images\SitemapConfigure.png
+	EndProjectSection
```

Comment:
We do not include docs and other files that you included in our solutions. 
Please undo .sln changes.

P.S. We might change that policy, but then we will change it in all our projects at once.